### PR TITLE
Lock out lines from the  SLD

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

**Comment by FranckLecuyer** [comment]
The popup menu for the lines is not always totally visible : 

lines_menu

**Comment by FranckLecuyer** [comment]
It is indicated in the US : 
When a line is currently locked/tripped, and the state of a switch at one end is being changed, Here's the desired behavior:
    Removal of the "locked/tripped" tag and icon, as if the line was only energize on one end (e.g. the video included in the attachment tab)

We still have the icons, after manually changing the state of a switch, after a trip/lockout :

File: src/translations/fr.json [hunk 1]
```diff
@@ -217,5 +217,15 @@
     "StopComputation": "Arrêter le calcul",
     "ColumnsList": "Liste des colonnes",
     "LabelSelectList": "Configurer colonnes",
-    "CheckAll": "Sélectionner tout / aucun"
+    "CheckAll": "Sélectionner tout / aucun",
+
+    "LockoutLine": "Consigner la ligne",
+    "TripLine": "Déclencher la ligne",
+    "EnergiseOnOneEnd": "Mettre sous tension à vide depuis {substation}",
```
Comment (FranckLecuyer) [inline src/translations/fr.json:224]: ({substation}) instead ? like in english translation

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/images/bolt_black_24dp.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><g><rect fill="none" height="24" width="24"/></g><g><path d="M11,21h-1l1-7H7.5c-0.88,0-0.33-0.75-0.31-0.78C8.48,10.94,10.42,7.54,13.01,3h1l-1,7h3.51c0.4,0,0.62,0.19,0.4,0.66 C12.97,17.55,11,21,11,21z"/></g></svg>
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply src/images/bolt_black_24dp.svg:1]: Done

File: src/images/lock_black_24dp.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><g fill="none"><path d="M0 0h24v24H0V0z"/><path d="M0 0h24v24H0V0z" opacity=".87"/></g><path d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"/></svg>
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply src/images/lock_black_24dp.svg:1]: Done

**Comment by SlimaneAmar** [comment]
> The popup menu for the lines is not always totally visible :
> 
> lines_menu

File: src/translations/fr.json (hunk 1 above)
Comment (SlimaneAmar) [reply src/translations/fr.json:224]: This concerns the following PR :
https://github.com/gridsuite/gridstudy-app/pull/320
@niconoir

File: src/index.css [hunk 4]
```diff
@@ -383,4 +383,4 @@ polyline {
 .sld-label {
     fill: black;
     stroke: none;
-}
+}
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply src/index.css:386]: Done

**Comment by SlimaneAmar** [comment]
> It is indicated in the US :
> When a line is currently locked/tripped, and the state of a switch at one end is being changed, Here's the desired behavior:
> Removal of the "locked/tripped" tag and icon, as if the line was only energize on one end (e.g. the video included in the attachment tab)
> 
> We still have the icons, after manually changing the state of a switch, after a trip/lockout :
> line

**Comment by SlimaneAmar** [comment]
> It is indicated in the US :
> When a line is currently locked/tripped, and the state of a switch at one end is being changed, Here's the desired behavior:
> Removal of the "locked/tripped" tag and icon, as if the line was only energize on one end (e.g. the video included in the attachment tab)
> 
> We still have the icons, after manually changing the state of a switch, after a trip/lockout :
> line

An US has been created :
https://dev.azure.com/GridSuite/MVP/_workitems/edit/520

**Comment by SlimaneAmar** [comment]
> The popup menu for the lines is not always totally visible :
> 
> lines_menu

Corrected

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
The correction done in PR : https://github.com/gridsuite/gridstudy-app/pull/339
(PR merged in master on Friday 04/06)

has been lost ....

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**